crates/store/re_types_core/src/reflection.rs

```diff
@@ -238,6 +238,11 @@ pub struct ArchetypeReflection {
     /// e.g. `Spatial3DView`.
     pub view_types: &'static [&'static str],
 
+    /// Does this have a particular scope?
+    ///
+    /// e.g. `"blueprint"`.
+    pub scope: Option<&'static str>,
```

Comment:
Really useful to filter stuff at runtime, without having to rely on shady `str::contains` shenanigans.